ex2/ex2.js

```diff
@@ -8,3 +8,24 @@
  *     fn(",,,", 4) === -1
  *     fn("hel,lowo,rld", 0) - Throws
  */
+function getTheCommaIndexAt(str, commaIndex) {
+
+	if (commaIndex === 0) {
```

Comment:
Before you were checking for negative numbers (-3, for example) too and non numbers. Please put these back.